packages/node_modules/@webex/plugin-meetings/src/member/index.js

```diff
@@ -0,0 +1,174 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+import {
+  MEETINGS,
+  IN_LOBBY,
+  NOT_IN_MEETING,
+  IN_MEETING
+} from '../constants';
+
+import MemberUtil from './util.js';
+
+/**
+ * @class Member
+ */
+export default class Member {
+  namespace = MEETINGS;
+
+  /**
+   * @param {Object} participant - the locus participant
+   * @param {Object} options - optional constructor params
+   * @returns {Member}
+   * @memberof Member
+   * shallow copies locus participant object into a meeting member object
+   */
+  constructor(participant, options = {}) {
+    this.participant = null;
+    this.id = null;
+    this.name = null;
+    this.isAudioMuted = null;
+    this.isVideoMuted = null;
+    this.isSelf = null;
+    this.isHost = null;
+    this.isGuest = null;
+    this.isInLobby = null;
+    this.isInMeeting = null;
+    this.isNotInMeeting = null;
```

Comment:
I feel like we only need the `isIn` cases and devs can `!` on them

Reply:
yeah I'm fine with that, some of those ideas were carry over from web client, and it felt nice to have just a straight boolean to read from, but i can change.